compiler-rt/lib/rtsan/rtsan.cpp

```diff
@@ -44,9 +45,15 @@ static InitializationState GetInitializationState() {
       atomic_load(&rtsan_initialized, memory_order_acquire));
 }
 
+static void RtsanAtexit() { PrintStatisticsSummary(); }
+
 static auto OnViolationAction(DiagnosticsInfo info) {
   return [info]() {
-    __rtsan::PrintDiagnostics(info);
+    if (flags().print_stats_on_exit)
+      IncrementTotalErrorCount();
```

Comment:
Is it more costly to always do the check, or just always increment a relaxed atomic?

Reply:
My guess would be that increment is cheaper than a branch but I don't think it's worth spending time on.

Reply:
removed the if, if we are even on the fence i'd opt to clean up the control flow.

Thanks for your feedback on this one, submitting